style.css

```diff
@@ -122,3 +127,210 @@ header {
 .social {
   height: 18px;
 }
+
+/* Works section */
+
+.works {
+  padding-bottom: 30px;
+}
+
```

Comment:
- [ ] The works section currently has a `white` background whereas the Figma design is `#E5E5E5`. Please effect this change